### PR TITLE
fix service not working when a node's IPv6 address is before the IPv4 address

### PR DESCRIPTION
**Pull request by lynn901**
- [ ] Make sure you have followed [Kube-OVN Code Style](../CODE_STYLE.md).

#### What type of this PR
- Bug fixes

fix service not working when a node's IPv6 address is before the IPv4 address
#### Which issue(s) this PR fixes:
Fixes #1721

### CODE REVIEW

**Comment by zhangzujian** [comment]
You can force push to the source branch. No need to create a new PR.

**Comment by lynn901** [comment]
> You can force push to the source branch. No need to create a new PR.

I will do this next time

**Comment by oilbeater** [comment]
@lynn901 please check the build error https://github.com/kubeovn/kube-ovn/runs/7442921444?check_suite_focus=true#step:9:18

**Comment by lynn901** [comment]
> @lynn901 please check the build error https://github.com/kubeovn/kube-ovn/runs/7442921444?check_suite_focus=true#step:9:18

have fix the error

**Comment by zhangzujian** [comment]
Function `CheckProtocol()` should also be modified to support format `<IPv6>,<IPv4>`.

File: pkg/util/net.go [hunk 1]
```diff
@@ -303,8 +303,13 @@ func GetIpWithoutMask(ipStr string) string {
...
+		for _, ipTmp := range strings.Split(ipStr, ",") {
+			if CheckProtocol(ipTmp) == kubeovnv1.ProtocolIPv4 {
+				v4IP = ipTmp
+			} else {
+				v6IP = ipTmp
+			}
+		}
 	} else if CheckProtocol(ipStr) == kubeovnv1.ProtocolIPv4 {
 		v4IP = ipStr
 	} else if CheckProtocol(ipStr) == kubeovnv1.ProtocolIPv6 {
```
Comment (zhangzujian) [inline pkg/util/net.go:315]: The if-else conditions can be replaced with switch-case:

```go
switch CheckProtocol(ipStr) {
    case kubeovnv1.ProtocolIPv4:
        // do sth
    case kubeovnv1.ProtocolIPv6:
        // do sth
    ...
}
```

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**